src/function/algebra/simplify.js

```diff
@@ -636,9 +643,16 @@ export const createSimplify = /* #__PURE__ */ factory(name, dependencies, (
       }
     } else if (rule instanceof SymbolNode) {
       // If the rule is a SymbolNode, then it carries a special meaning
-      // according to the first character of the symbol node name.
-      // c.* matches a ConstantNode
-      // n.* matches any node
+      // according to the first character or two of the symbol node name.
```

Comment:
I'm not sure if the list below helps, it's duplicate of what is in the top comment now. Maybe we can simply remove the following explanation?

Reply:
ping @thatcomputerguy0101 what do you think of my comment?

Reply:
Personally, when adding the new wildcard types, I found it annoying to scroll between the top and middle of this long document. This was minimized by opening the same file twice side-by-side, but I still think it is useful to have an explanation of the nodes close to where they are defined. Besides file size of the source code and having to update both locations, is there any downside to having extra comments?

Reply:
It's not a _big_ deal, but in general, we developers tend to forget to update comments after making changes in the code. So it can get outdated. Having two places with the same extensive comments is even more susceptible to being forgotten to update over time 😅 . I prefer having documentation in one place in general to prevent the docs from getting out of sync.

Reply:
On further consideration, the comments already in the switch cases for each rules serve the purposes that I mentioned in my previous message, so this section can just be referenced those.

Reply:
👍 

The inline comments in the switch are really helpful!